packages/mobx-undecorate/cli.js

```diff
@@ -13,6 +13,7 @@ spawn(
     ],
     {
         cwd: __dirname,
-        stdio: "inherit"
+        stdio: "inherit",
+        shell: true
```

Comment:
can you elaborate on why `shell` is needed here?

Reply:
@mweststrate I have no clue except that it didn't work without that. Found it on a StackOverflow as a comment somewhere while trying to figure it out. 
https://nodejs.org/api/child_process.html#child_process_child_process_spawn_command_args_options

Reply:
Tested this with normal cmd, powershell, git-bash, neither worked without the shell: true set.